control/discovery.py

```diff
@@ -717,16 +731,9 @@ def reply_get_log_page(self, conn, data, cmd_id):
             log_entry_counter = 0
             while log_entry_counter < len(allow_listeners):
                 log_entry = DiscoveryLogEntry()
-                log_trtype = allow_listeners[log_entry_counter]["trtype"]
-                log_adrfam = allow_listeners[log_entry_counter]["adrfam"]
-                trtype = GatewayEnumUtils.get_value_from_key(pb2.TransportType, log_trtype, True)
-                adrfam = GatewayEnumUtils.get_value_from_key(pb2.AddressFamily, log_adrfam, True)
-
-                if trtype is None:
-                    self.logger.error(f"unsupported transport type {log_trtype}")
-                else:
-                    log_entry.trtype = trtype
-
+                log_entry.trtype = TRANSPORT_TYPES.TCP
+                log_adrfam = allow_listeners[log_entry_counter]["adrfam"].lower()
```

Comment:
nit: .lower() is not required, since spdk uses strcasecmp(), see [here](https://github.com/spdk/spdk/blob/66ec2569b523b88ae320b94b6a0da52ae1ff8de6/lib/nvme/nvme.c#L1122-L1125).